spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/RefBuilder.scala

```diff
@@ -0,0 +1,52 @@
+package com.datastax.spark.connector.util
+
+case class ConfigParameter (
+  val name:String,
+  val section: String,
+  val default:Option[Any],
```

Comment:
This is a good idea :) But will require me to rewrite a lot of things, starting now :)